.github/workflows/test-suite.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches: ["master"]
   pull_request:
     branches: ["master"]
+  workflow_dispatch:
```

Comment:
Why did you add this?

Reply:
This enables dev branches to manually run the test suite. 

![image](https://user-images.githubusercontent.com/24665/233801784-5d8d9cfa-7b81-4341-be9b-71f45220339b.png)

Reply:
Without creating a PR you mean?

Reply:
Yes, that's correct. It allows for running the test suite manually on your own fork or branch prior to creating a pull request.